Change_Flashers/Soave_Flasher_OOP.py

```diff
@@ -121,14 +66,12 @@ def func_opt(parametrs):
             liquid = CEOSLiquid(SRKMIX, eos_kwargs=eos_kwargs)
             flasher = FlashVL(constants2, properties, liquid=liquid, gas=gas)
 
-            PT = flasher.flash(T=T[i] + 273.15, P=P_bar[i][j] * pow(10, 5), zs=zs)
-            if PT.VF == 1:  # Есть только пар
-                print('Неудача')
-            else:  # Не только пар
+            PT = flasher.flash(T=T[i] + 273.15, P=pressure.values[i][j] * pow(10, 5), zs=zs)

```

Comment:
Why you call flasher. flash here, if you have  function for that purpose?
Again if PT.VF == 1:
we miss the point.
It is an error, we should add some big value to ABS_error, because the calculation contradicts the experiment.